File: go/p1/p1.go
package main

import "fmt"

func main() {
	ceiling := 1000
	sum := multiplesOfThreeOrFive(ceiling)

	fmt.Println(sum)
}

func multiplesOfThreeOrFive(ceiling int) int {
	y := 0

	for x := 3; x < ceiling; x++ {
		if x%3 == 0 || x%5 == 0 {
			y += x
		}
	}

	return y
}

File: py/p1/p1.py
def multiples_of_three_or_five(ceiling):
    y = 0
    for i in range(3, ceiling):
        if i % 3 == 0 or i % 5 == 0:
            y += i
    return y

if __name__ == '__main__':
    ceiling = 1000
    sum = multiples_of_three_or_five(ceiling)
    print(sum)

